pyproject.toml

```diff
@@ -1,25 +1,15 @@
 [tool.poetry]
 name = "adaptive_scheduler"
-version = "1.5.1"
+version = "1.6.0"
```

Comment:
Sorry to be a semver pedant, but incompatible API changes should bump the major version (first number). Probably doesn't matter because no one is using this as a library.

Reply:
You are undoubtedly right but I think I'm going to leave it at this since no one uses it like you said.